- name: "Create DB network"
  docker_network:
    name: "{{ DOCKER.net.dbms }}"

- community.docker.docker_container:
    name: mysql
    state: started
    image: mariadb:10.5 # Keeping at 10.5 because of https://help.nextcloud.com/t/update-to-next-cloud-21-0-2-has-get-an-error/117028/13
    pull: true
    recreate: no
    restart_policy: always
    volumes:
      - "{{ DOCKER.host_path }}/mysql:/var/lib/mysql"
    ports:
      - "3306:3306"
    env:
      MYSQL_ROOT_PASSWORD: "{{ SQL.root_password }}"
    purge_networks: yes
    networks:
      - name: "{{ DOCKER.net.dbms }}"
        aliases: [mysql]